flags.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+function realpath () {
+    echo $(cd "$(dirname "$1")"; pwd)/$(basename "$1")
+}
+
+function gccPath () {
+    echo $(cd $(dirname $(which gcc-7)) && echo $(dirname $(realpath $(readlink gcc-7))));
+}
+
+function cxxFlags () {
+    echo "-nostdinc++ -std=gnu++1z"
+    echo | g++-7 -Wp,-v -x c++ - -fsyntax-only 2>&1 | grep "^[[:space:]]" | while read path; do
```

Comment:
It would be great to generalize the `g++-7` command - could we store it in a variable called `GCC`? we could then make that the only variable a user has to set, depending on whether they used brew or macports

Reply:
sure, that makes sense.